tests/integration_tests/functional/test_api.py

```diff
@@ -469,13 +461,12 @@ def test_api_machine_config(test_microvm_with_api):
 
     assert test_microvm.api_session.is_status_no_content(response.status_code)
 
-    # Set the cpu template again
-    response = test_microvm.machine_cfg.patch(cpu_template="C3")
+    # Set the cpu template
     if platform.machine() == "x86_64":
-        assert test_microvm.api_session.is_status_no_content(response.status_code)
+        response = test_microvm.machine_cfg.patch(cpu_template="C3")
     else:
-        assert test_microvm.api_session.is_status_bad_request(response.status_code)
-        assert "CPU templates are not supported on aarch64" in response.text
+        response = test_microvm.machine_cfg.patch(cpu_template="None")
```

Comment:
Could we test an aarch64 static template instead?

Reply:
Not yet. At least no the one I have, because it requires patched 5.10 kernel. Maybe it would be worth creating some additional static template, but then what if someone will want to use it?